tools/openapi-plugin/src/generators/client/generator.ts

```diff
@@ -113,4 +122,27 @@ async function copySchema(
     }
 }
 
+async function validate(schemaPath: string) {
```

Comment:
Really all this validate function exists to do is spawn a child and run a cli command before the generator spins up. The reason being that I think it would be unwise to rely on the user running a manual check before attempting to generate. And generating based on a unvalidated schema will produce bad outcomes. That's the reasoning behind keeping it in generator code. 

Theres already a validate step which accomplishes the same task. An executor may be overkill here?